src/main/java/com/squareup/javapoet/MethodSpec.java

```diff
@@ -182,9 +185,13 @@ public static Builder overriding(ExecutableElement method) {
     String methodName = method.getSimpleName().toString();
     MethodSpec.Builder methodBuilder = MethodSpec.methodBuilder(methodName);
 
-    // TODO copy method annotations.
-    // TODO check to ensure we're not duplicating override annotation.
     methodBuilder.addAnnotation(Override.class);
+    for (AnnotationMirror mirror : method.getAnnotationMirrors()) {
+      if (((TypeElement) mirror.getAnnotationType().asElement()).getQualifiedName()
+          .equals(Override.class.getName()))
```

Comment:
Tried to insert them where needed. Missed those. But why didn't the Travis report the checkstyle problem?